hazelcast/src/main/java/com/hazelcast/map/impl/NearCache.java

```diff
@@ -93,21 +96,21 @@ public int compare(CacheRecord o1, CacheRecord o2) {
         }
     };
 
-    private SizeEstimator nearCacheSizeEstimator;
+    private final SizeEstimator nearCacheSizeEstimator;
```

Comment:
Is there any reason to keep this field duplicated too?  It could easily delegate to `mapContainer` too.